user_data/strategies/bb.py

```diff
@@ -0,0 +1,153 @@
+# --- Do not remove these libs ---
+from pandas import DataFrame
+# --------------------------------
+
+import talib.abstract as ta
+import freqtrade.vendor.qtpylib.indicators as qtpylib
+import numpy  # noqa
+from wao.wao_strategy import WAOStrategy
+
+
+class bb(WAOStrategy):
```

Comment:
do not add a new file, the task is to update the parameters in the existing strategy, keep the name as bbrsi_scalp for now